core/data.go

```diff
@@ -40,6 +40,17 @@ type Blob struct {
 	Data          []byte
 }
 
+type BlobAuthHeader struct {
+	// Commitments
+	BlobCommitments `json:"commitments"`
+	// AccountID is the account that is paying for the blob to be stored
+	AccountID AccountID `json:"account_id"`
+	// Nonce
+	Nonce uint32 `json:"nonce"`
```

Comment:
does it need to be incremented for every call? looks like `Nonce` is just being used as a message to sign

Reply:
Eventually this will be an incremental nonce (https://docs.google.com/document/d/1w2H38txbe21ex8bwxNMAcjRq4K5WHq0NCv6jYy8yo54/edit#heading=h.2ty1ce44eg7b). However, since we don't yet support payments we can simplify things. I think that the random challenge approach is the quickest way to get us the minimal capability that we want.